lib/util.js

```diff
@@ -1,16 +1,16 @@
 'use strict';
 
 exports.getValueAsString = function getValueString(value) {
+	if (Number.isFinite(value)) {
+		return `${value}`;
+	}
+
 	if (Number.isNaN(value)) {
 		return 'Nan';
-	} else if (!Number.isFinite(value)) {
-		if (value < 0) {
-			return '-Inf';
-		} else {
-			return '+Inf';
-		}
+	} else if (value < 0) {
+		return '-Inf';
 	} else {
-		return `${value}`;
+		return '+Inf';
 	}
```

Comment:
Fine as-is, but you could also remove all of the `else`s to possibly shorten this fn a bit.

Can you update the changelog please? LGTM otherwise.